### ISSUE TITLE
How to use this plugin

### ISSUE THREAD

**Issue by sifrah** [jenkinsci/sidebar-link-plugin#61]
### Describe your use-case which is not covered by existing documentation.

I would like to put a link or button in the navbar.
I've installed the plugin without any issues, but i'm unable to use the plugin.

I'm started from 0 

<img width="1278" alt="image" src="https://github.com/jenkinsci/sidebar-link-plugin/assets/6054235/5c1ce219-1dc2-4372-b50e-f8fe1e724b78">

**Comment by damianszczepanik** [comment]
True. You can find configuration in Manage Jenkins section - same place for this and other plugins
